src/components/generic/InputField/InputField.scss

```diff
@@ -21,7 +22,7 @@ $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-input-field;
     width: 100%;
 
     &[type="number"] {
```

Comment:
is this still necessary? I would prefer not to format another component from this component like this